ivy_tests/test_ivy/test_frontends/test_tensorflow/test_raw_ops.py

```diff
@@ -622,6 +622,41 @@ def test_tensorflow_Atan(  # NOQA
     )
 
 
+# Atan2
+@handle_frontend_test(
+    fn_tree="tensorflow.raw_ops.Atan2",
+    dtype_and_y=helpers.dtype_and_values(
```

Comment:
Here we can pass the `num_arrays` parameter instead of calling the helper function twice. this can generate both `x` and `y`

Reply:
done